demos/text_detection_demo/include/cnn.hpp

```diff
@@ -14,7 +14,7 @@ using namespace InferenceEngine;
 
 class Cnn {
   public:
-    Cnn():is_initialized_(false), channels_(0), input_data_(nullptr), time_elapsed_(0), ncalls_(0) {}
+    Cnn():is_initialized_(false), channels_(0), input_name_(""), input_data_(nullptr), time_elapsed_(0), ncalls_(0) {}
```

Comment:
This initializer is unnecessary; `input_name_` would get initialized to an empty string anyway.